src/OrbitGl/App.cpp

```diff
@@ -2191,8 +2202,10 @@ bool OrbitApp::IsCapturing() const {
   return capture_client_ != nullptr && capture_client_->IsCapturing();
 }
 
+bool OrbitApp::IsLoadingCapture() const { return is_loading_capture_; }
+
 bool OrbitApp::IsCapturingOrLoading() const {
-  return is_capture_loading_ || (capture_client_ != nullptr && capture_client_->IsCapturing());
+  return is_loading_capture_ || (capture_client_ != nullptr && capture_client_->IsCapturing());
```

Comment:
do we need this here? I added it for interactivity which is now disabled. I think we can drop IsCapturingOrLoading and make IsCapturing public

Reply:
Done.